psutil_agent/tests/test_loop.py

```diff
@@ -0,0 +1,65 @@
+import sched
+import time
+
+import mock
+import testtools
```

Comment:
are you using anything testtools specific? im just wondering because we usually use unittest.TestCase